lib/python/rose/macro.py

```diff
@@ -1335,7 +1335,7 @@ def _report_error(exception=None, text=""):
     )
 
 
-def scan_rose_directory(conf_dir):
+def scan_rose_directory(conf_dir, top_level_only=False):
```

Comment:
Should `top_level_only` be `suite_only`? (Or vice versa.) Using the same variable name makes it easier to search for the functionality in the future.